src/patternfly/components/Button/styles.scss

```diff
@@ -118,6 +109,8 @@
 
   position: relative;
   display: inline-block;
+  min-width: var(--pf-c-button--MinWidth);
+  min-height: var(--pf-c-button--MinWidth);
```

Comment:
This is unrelated to your work but can you update this to be --pf-c-button--minHeight. It was missed on a previous review.